layers/mldv1.go

```diff
@@ -0,0 +1,174 @@
+// Copyright 2018 Nine Internet Solutions AG. All rights reserved.
```

Comment:
Would you be okay to modify this to say "Copyright 2018 GoPacket Authors" and put your (and/or your company's) name in the top-level AUTHORS file?

Reply:
in the `AUTHOR` file, what are 'Authors' and what are 'Contributors'?

Reply:
Somewhat subjective, but I'd say "author" is more than 25 lines of code.  So you're an author ;)